apps/sms/js/thread_list_ui.js

```diff
@@ -393,7 +393,8 @@ var ThreadListUI = {
     }
   },
 
-  markReadUnread: function thlui_markReadUnread(selected, isRead) {
+  markReadUnreadUI: function thlui_markReadUnreadUI(selected, isRead) {
+    var threadToMark = [];
     selected.forEach((id) => {
```

Comment:
@julienw thanks for introducing to these awesome methods :) where were these methods hidden? ;)